variables.tf

```diff
@@ -338,3 +338,9 @@ variable "postrender_binary_path" {
   description = "Relative or full path to command binary."
   default     = null
 }
+
+variable "permissions_boundary" {
+  type        = string
+  description = "If provided, IAM role will be created with this permissions boundary attached"
```

Comment:
Please use the same description

https://github.com/cloudposse/terraform-aws-eks-iam-role/blob/b87373712671560e774b9733004111795495c377/variables.tf#L46

Reply:
Fixed